gamemanager.cpp

```diff
@@ -0,0 +1,34 @@
+#include "gamemanager.h"
+
+GameManager::GameManager(){
+
+}
+
+GameManager* GameManager::Instance() {
```

Comment:
minek csinalsz singleton patternt? kicsit overkillnek tunik ez itt.

Reply:
Ez tovabbra is erdekelne.

Reply:
azért akartam ilyet használni, mert akkor ebből az osztályból csak egy példány létezik és itt el lehetne tárolni például a karaktereket meg több mindent is, hátha a jövőben lesz majd ilyen. De itt még tényleg semmi értelme.